python/ray/tests/BUILD

```diff
@@ -129,6 +129,7 @@ py_test_module_list(
 py_test_module_list(
   files = [
     "test_failure.py",
+    "test_failure2.py",
```

Comment:
Make them medium tests?

Reply:
Hmm maybe we can change the name of the job, but isn't it better to leave it in this job so that the work is more evenly distributed between jobs?